types/index.d.ts

```diff
@@ -0,0 +1,11 @@
+export type Transaction = Array<object>;
+export type Schema = object;
+export type config = {schema?: Schema, initialData?: Transaction };
+export type Entity = {
+  get: (...params:string[]) => any;
+}
+
+export function HomebaseProvider(props: {config?:config, children:any}): any;
+export function useTransact(): [(transaction:Transaction) => any];
```

Comment:
@smothers Update from when we paired: I had to name the transaction to get this to work